nwss_wastewater/delphi_nwss/run.py

```diff
@@ -146,7 +146,7 @@ def run_module(params):
         df = generate_weights(df, sensor)
 
         for geo in GEOS:
-            logger.info("Generating signal and exporting to CSV", metric=sensor)
+            logger.info("Generating signal and exporting to CSV", geo=geo, sensor=sensor)
```

Comment:
great catch on the previously missing geo!  lets call it `signal` for consistency tho:
```suggestion
            logger.info("Generating signal and exporting to CSV", geo=geo, signal=sensor)
```